### PR TITLE
added a new input time.initial_dt

### PR DESCRIPTION
**Pull request by michaeljbrazell**
- added an initial_dt input parameter to help startup with CFL controlled runs
- added documentation 
- changed time.use_force_cfl default to true

### CODE REVIEW

File: docs/sphinx_doc/user/inputs.rst [hunk 1]
```diff
@@ -153,6 +153,16 @@ This section deals with parameters that control the simulation.
    Fixed timestep size (in seconds) used to advance the simulation. If this
    parameter is negative, then :input_param:`time.cfl` is used to determine the
    adaptive timestep during the simulation.
+   
+.. input_param:: time.initial_dt
+
+   **type:** Real number
+
+   Initial timestep size (in seconds) used to initialize the simulation. 
+   Only activated if :input_param:`time.cfl` = true and parameter is positive,
```
Comment (sayerhs) [inline docs/sphinx_doc/user/inputs.rst:162]: `= true` seems misleading, should just say 

```rst
if :input_param:`time.cfl` is a positive number
``` ?

Comment (michaeljbrazell) [reply docs/sphinx_doc/user/inputs.rst:162]: oh I meant something else, I'll fix it

File: docs/sphinx_doc/user/inputs.rst [hunk 2]
```diff
@@ -159,8 +159,7 @@ This section deals with parameters that control the simulation.
    **type:** Real number
 
    Initial timestep size (in seconds) used to initialize the simulation. 
-   Only activated if :input_param:`time.cfl` = true and parameter is positive, 
-   if this parameter is negative it is ignored.
+   Only activated if :input_param:`time.cfl` = true and if ``time.initial_dt`` is positive.
```
Comment (sayerhs) [inline docs/sphinx_doc/user/inputs.rst:162]: Is `time.cfl` boolean? Isn't it a real number?

Comment (michaeljbrazell) [reply docs/sphinx_doc/user/inputs.rst:162]: dang, I don't know what I was thinking, I meant if fixed_dt < 0. I'll fix it. good catch

**Pull request merged**